tests/resources/ci/scripts/setup.sh

```diff
@@ -69,7 +72,7 @@ installBaseSoftware() {
         installDockerOnLinux
     elif [[ $OS == "Darwin" ]]; then
         brew update
-        brew install curl unzip
+        brew install curl unzip || true
```

Comment:
How are we OK if curl and unzip don't install successfully?   Are we saying from the link to the setup-python issue that the error's happening because something unrelated like Python is failing to install...actually?

Reply:
The command in question fails because there is an update/upgrade that happens as result. The update/upgrade of associated software reports some errors. The errors are mentioned in the issue I referenced. Those errors cause the command to fail because we use set -e on purpose. In most cases, one or both of the commands we are trying to install will be already installed; however, we try to install them regardless to make sure they are available for the tests.  If curl or zip are not available, the test that use will fail when they are used.